webrender/src/device/gfx/render_pass.rs

```diff
@@ -113,6 +134,18 @@ impl<B: hal::Backend> HalRenderPasses<B> {
                 .. hal::image::Layout::DepthStencilAttachmentOptimal,
         };
 
+        let attachment_depth_clear = hal::pass::Attachment {
+            format: Some(depth_format),
+            samples: 1,
+            ops: hal::pass::AttachmentOps::new(
+                hal::pass::AttachmentLoadOp::Clear,
+                hal::pass::AttachmentStoreOp::Store,
```

Comment:
In a standard case where a render pass does all the rendering into a target, and depth is used from zero, we don't care about the resulting depth values in the end, so we should be using `DontCare` mode for this store operation.
This is something we'll need to follow-up with, no need to try to fit it in this PR.